moped-editor/src/views/projects/projectView/ProjectPhases.js

```diff
@@ -1,411 +1,232 @@
+        field: "moped_phase",
+        minWidth: 200,
+        valueGetter: ({ row }) => row.moped_phase?.phase_name,
+      },
+      {
+        headerName: "Subphase",
+        field: "moped_subphase",
+        minWidth: 200,
+        valueGetter: ({ row }) => row.moped_subphase?.subphase_name,
+      },
+      {
+        headerName: "Start",
+        field: "phase_start",
+        type: "date",
+        /** valueGetter is used by the date sort function inherently used by the `date` type column */
+        valueGetter: ({ row }) =>
+          row.phase_start ? new Date(row.phase_start) : null,
+        /**  the renderCell function controls the react node rendered for this cell */
+        renderCell: ({ row }) => {
+          let strToRender = row.phase_start
+            ? new Date(row.phase_start).toLocaleDateString()
+            : "";
+
+          const showNotConfirmedIndicator =
+            !row.is_phase_start_confirmed && strToRender;
+
+          strToRender = showNotConfirmedIndicator
+            ? `${strToRender}*`
+            : strToRender;
+          return (
+            <ProjectPhaseDateConfirmationPopover
+              isEnabled={showNotConfirmedIndicator}
+              dateType="start"
+            >
+              <span>{strToRender}</span>
+            </ProjectPhaseDateConfirmationPopover>
+          );
+        },
+        minWidth: 150,
+      },
+      {
+        headerName: "End",
+        field: "phase_end",
+        type: "date",
+        /** valueGetter is used by the date sort function inherently used by the `date` type column */
+        valueGetter: ({ row }) =>
+          row.phase_end ? new Date(row.phase_end) : null,
+        /**  the renderCell function controls the react node rendered for this cell */
+        renderCell: ({ row }) => {
+          let strToRender = row.phase_end
+            ? new Date(row.phase_end).toLocaleDateString()
+            : "";
+
+          const showNotConfirmedIndicator =
+            !row.is_phase_end_confirmed && strToRender;
+
+          strToRender = showNotConfirmedIndicator
+            ? `${strToRender}*`
+            : strToRender;
+          return (
+            <ProjectPhaseDateConfirmationPopover
+              isEnabled={showNotConfirmedIndicator}
+              dateType="end"
+            >
+              <span>{strToRender}</span>
+            </ProjectPhaseDateConfirmationPopover>
+          );
+        },
+        minWidth: 150,
+      },
+      {
+        headerName: "Description",
+        field: "phase_description",
+        minWidth: 350,
+      },
+      {
+        headerName: "Current",
+        field: "is_current_phase",
+        minWidth: 50,
+        renderCell: ({ row }) =>
+          row.is_current_phase ? (
+            <Box display="flex">
+              <CheckCircleOutline style={{ color: green[500] }} />
+            </Box>
+          ) : (
+            ""
+          ),
+      },
+      {
+        headerName: "",
+        field: "_edit",
```

Comment:
**Possible change requested** I noticed that this field name shows up when using the "Manage Columns" menu to show/hide columns. Not sure if this is a big deal or even able to be changed.
<img width="281" alt="Screenshot 2024-01-16 at 4 21 35 PM" src="https://github.com/cityofaustin/atd-moped/assets/37249039/bdfe4e2b-8f37-41c2-b579-c23cff900546">

<img width="326" alt="Screenshot 2024-01-16 at 4 21 43 PM" src="https://github.com/cityofaustin/atd-moped/assets/37249039/2fe5be8f-5df2-4374-8cc9-7503ad4cbfeb">

Reply:
turns out we can toggle it! 

https://mui.com/x/react-data-grid/column-visibility/#customize-the-list-of-columns-in-panel

Reply:
thanks for catching this. i decided to just disabled this whole menu and the filters. so now it's only possible to sort.